src/components/widgets/LabelTextarea.vue

```diff
@@ -4,11 +4,16 @@ import { ref } from 'vue';
 defineEmits<{
   'update:modelValue': [value: string | undefined],
 }>();
-defineProps<{
+withDefaults(defineProps<{
   modelValue: string | null | undefined;
   label: string;
   error?: string;
-}>();
+  disabled?: boolean;
+}>(), {
+  error: '',
+  readonly: false
+});
+
```

Comment:
Why not adding `disabled` prop with defaults?

Reply:
That's because I meant to rename the property in defaults from `readonly` to `disabled` as well and didn't do it :smile: Thanks!